packages/docregistry/src/registry.ts

```diff
@@ -1347,71 +1360,81 @@ export namespace DocumentRegistry {
         displayName: trans.__('Markdown File'),
         extensions: ['.md'],
         mimeTypes: ['text/markdown'],
-        icon: markdownIcon
+        icon: markdownIcon,
+        collaborative: true
       },
       {
         name: 'PDF',
         displayName: trans.__('PDF File'),
         extensions: ['.pdf'],
         mimeTypes: ['application/pdf'],
-        icon: pdfIcon
+        icon: pdfIcon,
+        collaborative: true
       },
       {
         name: 'python',
         displayName: trans.__('Python File'),
         extensions: ['.py'],
         mimeTypes: ['text/x-python'],
-        icon: pythonIcon
+        icon: pythonIcon,
+        collaborative: true
       },
       {
         name: 'json',
         displayName: trans.__('JSON File'),
         extensions: ['.json'],
         mimeTypes: ['application/json'],
-        icon: jsonIcon
+        icon: jsonIcon,
+        collaborative: true
       },
       {
         name: 'julia',
         displayName: trans.__('Julia File'),
         extensions: ['.jl'],
         mimeTypes: ['text/x-julia'],
-        icon: juliaIcon
+        icon: juliaIcon,
+        collaborative: true
       },
       {
         name: 'csv',
         displayName: trans.__('CSV File'),
         extensions: ['.csv'],
         mimeTypes: ['text/csv'],
-        icon: spreadsheetIcon
+        icon: spreadsheetIcon,
+        collaborative: true
       },
       {
         name: 'tsv',
         displayName: trans.__('TSV File'),
         extensions: ['.tsv'],
         mimeTypes: ['text/csv'],
-        icon: spreadsheetIcon
+        icon: spreadsheetIcon,
+        collaborative: true
       },
       {
         name: 'r',
         displayName: trans.__('R File'),
         mimeTypes: ['text/x-rsrc'],
         extensions: ['.R'],
-        icon: rKernelIcon
+        icon: rKernelIcon,
+        collaborative: true
       },
       {
         name: 'yaml',
         displayName: trans.__('YAML File'),
         mimeTypes: ['text/x-yaml', 'text/yaml'],
         extensions: ['.yaml', '.yml'],
-        icon: yamlIcon
+        icon: yamlIcon,
+        collaborative: true
       },
       {
         name: 'svg',
         displayName: trans.__('Image'),
         mimeTypes: ['image/svg+xml'],
         extensions: ['.svg'],
         icon: imageIcon,
-        fileFormat: 'base64'
+        fileFormat: 'base64',
+        collaborative: true
       },
       {
         name: 'tiff',
```

Comment:
I explicitly chose not to enable collaboration on the image formats below. Let me know if you don't think this makes sense.